### PR TITLE
EVEREST-692 Rename password command to token

### PR DESCRIPTION
**Pull request by recharte**
**Rename password command to token**
---
**Problem:**
EVEREST-692

Calling it password is incorrect as passwords are usually used in conjunction with a user name. This is a authorization token, let's call it that.

**Solution:**
Rename password command to token

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?

**Tests**
- ~[ ] Is an Integration test/test case added for the new feature/change?~
- ~[ ] Are unit tests added where appropriate?~

### CODE REVIEW

**Review by gen1us2k — APPROVED** [review]

File: commands/token.go [hunk 1]
```diff
@@ -20,15 +20,15 @@ import (
...
 	"go.uber.org/zap"
 
-	"github.com/percona/percona-everest-cli/commands/password"
+	"github.com/percona/percona-everest-cli/commands/token"
 )
 
 func newPasswordCmd(l *zap.SugaredLogger) *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "password",
+		Use: "token",
```
Comment (oksana-grishchenko) [inline commands/token.go:28]: Let's rename this function to `newTokenCmd` for consistency

File: cli-tests/tests/flow/all-operators.spec.ts [hunk 2]
```diff
@@ -50,7 +50,6 @@ test.describe('Everest CLI install', async () => {
         'percona-server-mongodb-operator operator has been installed',
         'percona-postgresql-operator operator has been installed',
         'everest-operator operator has been installed',
-        'Your new password is',
```
Comment (oksana-grishchenko) [inline cli-tests/tests/flow/all-operators.spec.ts:53]: Should we add a check for the new string here?

**Review by oksana-grishchenko — APPROVED** [review]

File: pkg/install/install.go [hunk 3]
```diff
@@ -125,16 +125,16 @@ func (o *Install) Run(ctx context.Context) error {
...
 		return errors.Join(err, errors.New("could not get the everest token secret"))
 	}
 	if err != nil && k8serrors.IsNotFound(err) {
-		pwd, err := o.generatePassword(ctx)
+		pwd, err := o.generateToken(ctx)
 		if err != nil {
 			return err
 		}
-		o.l.Info(pwd)
+		fmt.Println(pwd) //nolint:forbidigo
```
Comment (michal-kralik) [inline pkg/install/install.go:137]: What's the reason this was switched from logger to `fmt`?
Cli tries not use `fmt` in other places.

Comment (recharte) [reply pkg/install/install.go:137]: We received [feedback](https://perconacorp.slack.com/archives/C0545J2BEJX/p1702028560348799) that the string was easy to get lost among the other logs so we decided to change the format to better stand out from the logs.

File: cli-tests/tests/flow/all-operators.spec.ts (hunk 2 above)
Comment (recharte) [reply cli-tests/tests/flow/all-operators.spec.ts:53]: I initially added it here but then I thought it didn't really belong here as this test is only focused on the operators installation.

File: commands/token.go (hunk 1 above)
Comment (recharte) [reply commands/token.go:28]: Thank you, I missed this.

Comment (recharte) [reply commands/token.go]: Force pushed the changes.

File: cli-tests/tests/flow/all-operators.spec.ts (hunk 2 above)
Comment (recharte) [reply cli-tests/tests/flow/all-operators.spec.ts:53]: I think that if we want to properly test the installation we should validate the created k8s objects, just checking the log is not that useful. We didn't even test if the printed token was actually correct. I think this is out of scope for this PR.

File: pkg/install/install.go (hunk 3 above)
Comment (michal-kralik) [reply pkg/install/install.go:137]: I suggest to keep this consistent.
`fmt` does not respect the `--json` flag.

`o.l.Info` can be for example prepended by new lines which would have the same effect.

Comment (recharte) [reply pkg/install/install.go]: Thanks, pushed https://github.com/percona/percona-everest-cli/pull/242/commits/3166cbf025debcdb8e08222722d1f041f3403313

**Pull request merged**